server/job-workers/job-worker-server.h

```diff
@@ -47,6 +49,8 @@ class JobWorkerServer : vk::not_copyable {
   bool reply_is_expected() const noexcept;
 
 private:
+  static const int EPOLL_FLAGS;
```

Comment:
constexpr static? Maybe place this var directly in source file?

Reply:
ok, done